### ISSUE TITLE
State Management / GraphQL Client for Labs #21

### ISSUE THREAD

**Issue by Vyraal1** [Lambda-School-Labs/mission-control-fe#224]
The current version of Mission Control (v1.0.0 per readme) uses React and URQL as it's GraphQL client. Labs #19 worked with URQL as it was passed down from Labs #18 with the reasoning it was lighter, and not as large as Apollo client. 

At the time, we didn't have an [argument for/against URQL](https://dev.to/open-graphql/graphql-and-urql-by-example-1ldc) so we went with it, which led to state management problems as eventually realized that URQL didn't have local state management, case and point, the monolithic component that is the `NoteEditor`, which utilizes are pretty hefty props API to manage it's local state. 

**Conclusion**
You can close this issue whenever you make the determination whether to use ContextAPI and React Hooks in the redux-style, or switching to Apollo Client for it's local state management features. 

tldr; Decide if you want to continue using URQL and another state management solution, or go with Apollo client
